### ISSUE TITLE
Emulate some more book 1 instructions

### ISSUE THREAD

**Issue by antonblanchard** [linuxppc/linux#101]
There are some new book 1 non vector instructions we should emulate in arch/powerpc/lib/sstep.c:

```
cmpb
bpermd
popcntb
popcntd
popcntw
prtyd
prtyw
isel
iseleq
iselgt
isellt
```

**Comment by m-brow** [comment]
All of these instructions have been added as of 4.14-1.

**Comment by mpe** [comment]
Implemented as of e27f71e5ff3c, thanks @m-brow :beer:
